xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-webjar/src/main/webjar/lightbox.js

```diff
@@ -301,13 +305,17 @@ define('xwiki-lightbox', [
         $('#lightboxDownload').attr('download', imageData.fileName);
       }
     };
-    myOpenLightbox = gallery(media, options);
+    _openedLightbox = gallery(_slidesData, options);
   };
 
   /**
    * Initialize the lightbox functionality for a set of images.
    */
   var initLightboxFunctionality = function() {
+    // The lightbox will be added to xwikicontent images that don't explicitly disable it.
+    _lightboxImages = $('#xwikicontent img')
+      .filter((i, img) => $(img.closest('[data-xwiki-lightbox]')).data('xwiki-lightbox') != false);
```

Comment:
jQuery documentation seems to indicate https://api.jquery.com/filter/#filter-function that the second parameter of the filter function is of type ``Element`` so I don't understand how you can call ``closest`` on it? Maybe you wanted:

```suggestion
      .filter((i, img) => $(img).closest('[data-xwiki-lightbox]').data('xwiki-lightbox') != false);
```

Reply:
There is a [`Element.closest()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)